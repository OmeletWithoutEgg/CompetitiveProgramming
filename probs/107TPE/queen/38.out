-83604.42 -28724144.70
