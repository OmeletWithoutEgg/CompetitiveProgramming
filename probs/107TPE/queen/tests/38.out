-40445.00 -54209619.00
